packages/web/src/components/nav/desktop/LeftNavLink.tsx

```diff
@@ -1,58 +1,102 @@
 import { ComponentProps } from 'react'
 
-import cn from 'classnames'
+import { Text, TextProps, spacing } from '@audius/harmony'
+import { CSSInterpolation } from '@emotion/css'
+import { Interpolation, Theme } from '@emotion/react'
+import { Slot } from '@radix-ui/react-slot'
 import { NavLink, NavLinkProps } from 'react-router-dom'
-import { SetOptional } from 'type-fest'
-
-import { Droppable, DroppableProps } from 'components/dragndrop'
-import { selectDragnDropState } from 'store/dragndrop/slice'
-import { useSelector } from 'utils/reducer'
-
-import styles from './LeftNavLink.module.css'
 
 export type LeftNavLinkProps =
-  | { disabled?: boolean } & (
+  | { disabled?: boolean; asChild?: boolean } & (
       | Omit<NavLinkProps, 'onDrop'>
       | Omit<ComponentProps<'div'>, 'onDrop'>
     )
 
-export const LeftNavLink = (props: LeftNavLinkProps) => {
-  const { disabled, className: classNameProp, ...other } = props
-  const className = cn(classNameProp, styles.link, {
-    [styles.disabledLink]: disabled
-  })
+const indicatorCss: CSSInterpolation = {
+  content: '""',
+  display: 'block',
+  width: '20px',
+  height: '20px',
+  position: 'absolute',
+  top: 0,
+  bottom: 0,
+  margin: 'auto 0',
+  left: '-16px',
+  borderRadius: '4px'
+}
 
-  if ('to' in other) {
-    return <NavLink {...other} activeClassName='active' className={className} />
-  }
-  return <div {...other} className={className} />
+const linkInteractionCss: CSSInterpolation = {
+  '&:hover': {
+    cursor: 'pointer',
+    color: 'var(--harmony-n-950)'
+  },
+  '&:hover:before': [
+    indicatorCss,
+    {
+      borderRight: '4px solid var(--harmony-n-400)'
```

Comment:
4px can we use cornerRadius theme value?

Reply:
Did not know that was a thing!